doc/guide/Guide_Review.ipynb

```diff
@@ -2,10 +2,58 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "commercial-participant",
    "metadata": {},
    "source": [
-    "# Reviewer Checklist\n",
+    "# Reviewer Guidelines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to review a pull request\n",
+    "\n",
+    "- Be friendly\n",
+    "- Read and follow the [**Reviewer Checklist**](../guide/Guide_Reviewer_Checklist.ipynb)\n",
+    "- Decide how much time you can spare and the detail you can work in. Tell the author!\n",
+    "- Use the comment/chat functionality within GitHub's pull requests - it's useful to have an archive of discussions and the decisions made.\n",
+    "- Fix the big things first! If there are more important issues, not every style guide has to be stuck to,\\\n",
+    "  not every slight increase in speed needs to be pointed out, and test coverage doesn't have to be 100%.\n",
+    "- Make it clear when a change is optional, or is a matter of opinion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At a minimum\n",
+    "- Make sure unit and integration tests are passing.\n",
+    "- (For complete modules) Run the tutorial on your local machine and check it does what it says it does\n",
+    "- Check everything is fully documented"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At least one reviewer needs to\n",
+    "- Review all the changes in the pull request. Read what it's supposed to do, check it does that, and make sure the logic is sound.\n",
+    "- Check that the code follows the CLIMADA style guidelines `#TODO: link`\n",
```

Comment:
Are the following suggested links suitable to refer to as the CLIMADA style guide?

- Check that the code follows the CLIMADA style guidelines [Python Dos and Don't](../guide/Guide_PythonDos-n-Donts.ipynb) and [Python performance tips and best practice for CLIMADA developers](../guide/Guide_Py_Performance.ipynb)